tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = {py27,py35}-django{18,19,110}
+envlist = {py27,py35}-django{18,19}-djangorestframework{32}-djangofilter{011}, {py27,py35}-django{110}-djangorestframework{35}-djangofilter{10}
```

Comment:
You don't need the `{}` around single values. You only need them if you're doing something comma-separated.

Also, this seems like a lot of different variations. If particular versions of DRF and django-filter are needed to be compatible with django 1.10, you can just specify them using the same `django110` factor, rather than giving them their own factors.